apiserver/facades/client/modelmanager/modelmanager.go

```diff
@@ -670,14 +676,14 @@ func (m *ModelManagerAPI) DumpModelsDB(args params.Entities) params.MapResults {
 	return results
 }
 
-// ListModels returns the models that the specified user
-// has access to in the current server.  Only that controller owner
+// ListModelSummaries returns models that the specified user
+// has access to in the current server.  Only the controller owner
```

Comment:
Since we do not have the concept of 'controller owner', maybe we should say here 'controller admin'?